src/app/content/highlights/actions.ts

```diff
@@ -64,3 +64,7 @@ export const receiveSummaryHighlights = createStandardAction('Content/Highlights
 export const receiveHighlightsTotalCounts = createStandardAction(
   'Content/receiveHighlightsTotalCounts'
 )<CountsPerSource, LocationFilters>();
+export const showConfirmationModal = createStandardAction('Content/Highlights/showConfirmation')<{
+  callback: (key: 'deny' | 'confirm') => any
```

Comment:
[window.confirm](https://developer.mozilla.org/en-US/docs/Web/API/Window/confirm) is returning a `bool`. I think we could unify this. Also, this should probably return `void`

Reply:
`resolve(boolean)` in `showConfirmation.tsx` is a `void` type callback. So we could leave it returning `void` or make `void | boolean` ?

Reply:
I meant something like this `callback(confirmed: boolean) => void`